tell the user to use their own name, not the example. (#44115)
kubernetes/website: content/en/docs/tutorials/hello-minikube.md

    kubectl config view
    ```

> 1. View application logs for a container in a pod. Note, the name passed here should be the one shown in the output from the `kubectl get pods` command.

Reviewer: ```suggestion
1. View application logs for a container in a pod. Make sure to replace `hello-node-5f76cf6ccf-br9b5` with the name of the pod from `kubectl get pods` command output.
```

Reviewer 2: If you are disagree the suggestions, you cannot mark it as resolved.

Author: > I'm happy with that

That is not disagreeing, that is saying that I'm happy with it.

Reviewer: > > I'm happy with that
> 
> That is not disagreeing, that is saying that I'm happy with it.

@digininja,
You need to push the updated branch with above suggested changes, then only we can mark it as resolved.
Please work on it and make the changes available in PR.

> 1. View application logs for a container in a pod. Make sure to replace `hello-node-5f76cf6ccf-br9b5` with the name of the pod from `kubectl get pods` command output.

Reviewer 2: ```suggestion
1. View application logs for a container in a pod. Make sure to replace
   `hello-node-5f76cf6ccf-br9b5` with the name of the pod from `kubectl get pods` command output.
```
We can follow the context style to wrap long lines.

Author: Would it be better to wrap it at the start of the sentence?

Reviewer 2: All right, that will be 3 lines.

Author: How do I edit your suggested version to suggest my change? I can only find a way to edit my original PR.

Reviewer 2: Edit it locally and run the following commands to submit again:

```
git add .
git commit --amend --no-edit
git push -f
```

Author: I just tried it in the browser, I don't think it ended up in the right place but I've made the change.

    ```

1. View application logs for a container in a pod.
   Make sure to replace `hello-node-5f76cf6ccf-br9b5` with the name of the pod from

>    `kubectl get pods` command output.

Reviewer 3: Hi @digininja .
What do you think about adding a note?

```
{{< note >}}
Replace `hello-node-5f76cf6ccf-br9b5` in the `kubectl logs` command with the name of the pod from the `kubectl get pods` command output.
{{< /note >}}
```

Reviewer 4: @digininja , what do you think of @kbhawkey 's [suggestion](https://github.com/kubernetes/website/pull/44115/files#r1434631864)?

Author: Yes, it looks fine to me.

Reviewer: @digininja,

> Yes, it looks fine to me.

Are you still working on @kbhawkey 's [suggestion](https://github.com/kubernetes/website/pull/44115/files#r1434631864)?
Please push the changes to PR to proceed further.

1. View application logs for a container in a pod.
   
   {{< note >}}
   Replace `hello-node-5f76cf6ccf-br9b5` in the `kubectl logs` command with the name of the pod from the `kubectl get pods` command output.
   {{< /note >}}

>    `kubectl get pods` command output.

Reviewer 2: Seem line 151 can be removed?

Author: I've removed it.
